### PR TITLE
Refactor gui

### PR DESCRIPTION
**Pull request by JaspervanTilburg**
Fixing the canvas gui bug and zoom in bug

### CODE REVIEW

File: src/main/java/graph/SequenceGraph.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ private void createColumns() {
             ArrayList<Integer> children = parent.getChildren();    // Get all children
             for (Integer child : children) {
                 this.getNode(child).incrementColumn(parent.getColumn());
-                System.out.println(this.getNode(child).getId());
+                //System.out.println(this.getNode(child).getId());
```
Comment (Jipje) [inline src/main/java/graph/SequenceGraph.java]: You can simply remove this laggy code.

File: src/main/java/gui/GraphDrawer.java [hunk 2]
```diff
@@ -60,6 +60,9 @@ private void initializeDrawableNodes() {
      * @param column The Column that has to be in the centre.
      */
     public void zoom(final double factor, final int column) {
+        if ((factor < 1 && zoomLevel < 2) || (factor > 1 && zoomLevel > columns.size())) {
+            return;
```
Comment (Jipje) [inline src/main/java/gui/GraphDrawer.java]: Small simple fix. I like it.

**Comment by Jipje** [comment]
Wait! Found a bug! http://imgur.com/a/XXvDF

**Comment by JaspervanTilburg** [comment]
How did this bug happen? I can't reproduce so it is hard for me to fix it

**Comment by Jipje** [comment]
That's odd, me neither...

**Comment by Jipje** [comment]
In that case, keep our eyes open for it, but it looks fine for now.

**Pull request merged**